portfolio/src/main/java/com/google/sps/servlets/RandomFacts.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
```

Comment:
This servlet functions in exactly the same way as the button previously did. I implemented it to practice the json concepts in the week 2 walkthrough.